sys/kern/kern_cons.c

```diff
@@ -525,6 +525,9 @@ cnputc(int c)
 	struct consdev *cn;
 	const char *cp;
 
+	if (cn_mute || c == '\0')
```

Comment:
why add check for NUL? Aren't NULs valid output sometimes?

Reply:
I did not add the check for null, it was already there.  I added the  `cm_mute` flag check only.

Reply:
OK. that makes sense to me then.